easybuild/easyblocks/t/tensorflow.py

```diff
@@ -423,14 +432,14 @@ def configure_step(self):
                 filtered_path = os.pathsep.join([p for fil in path_filter for p in path if fil not in p])
                 env.setvar(var, filtered_path)
 
-        wrapper_dir = os.path.join(tmpdir, 'bin')
-        use_wrapper = False
+        self.setup_build_dirs()
```

Comment:
@Flamefire Why not do this in the easyblock constructor, since it doesn't require that the build environment is set up yet?

Reply:
Because it is creating folders (in /tmp) which I see more fit for the configure step. Can't really avoid that as we need that unique folder